openssl-dynamic/src/main/c/ssl.c

```diff
@@ -106,7 +106,7 @@ struct TCN_bio_bytebuffer {
  * so that the return value for SSL_OP_FUTURE_WORKAROUND will only be
  * reported by versions that actually support that specific workaround.
  */
-static const jint supported_ssl_opts = 0
+static const jlong supported_ssl_opts = 0
```

Comment:
+1 for removal! also fyi `jlong` (64 bit) does not necessarily compatible with `long` (at least 32 bit)